frontend/src/layout/header/Search.tsx

```diff
@@ -107,8 +108,12 @@ export const SearchField: React.FC<SearchFieldProps> = ({ variant }) => {
                             height,
                             paddingLeft: 42,
                             paddingRight: 12,
-                            ":hover": { outline: "2px solid var(--grey80)" },
-                            ":focus": { outline: "2px solid var(--accent-color)" },
+                            ":hover": {
+                                borderColor: "var(--grey80)",
+                                outline: "1.5px solid var(--grey80)",
+                            },
+                            ":focus-visible": { borderColor: "var(--accent-color)" },
+                            ...focusStyle({ width: 1.5 }),
```

Comment:
I found another tiiiiiny problem with this :o  

![image](https://user-images.githubusercontent.com/7419664/223497384-bd6e15b4-cabf-461e-b8f2-bf51c8fff96b.png)

Having the outline and the border right next to one another unfortunately leads to some rendering bugs. I am talking about the 4 slightly bright pixels inside the border there. I have found a solution for this that makes the code even way simpler, so I think it's better for all cases:

```suggestion
                            ...focusStyle({ offset: -1 }),
```

This just puts the outline over the border. 

Whether you do the same trick for the hover outline... I don't care. For those two greys I wouldn't be able to notice the rendering bug.

Reply:
Oh that is clever, nice.

Reply:
Mh you only changed the one line in those 4 places, but you didn't remove the `":focus-visible": { borderColor: "var(--accent-color)" }` anywhere. I don't think that line is needed, right?

Reply:
I thought that would be the case, but then decided to actually check myself this time and found that the border does shine through at the edges. It's hardly noticeable, but I figured it's still better to keep the `borderColor`. (Sure, with the grey border we use at the moment it won't be an issue and I had to change the color to make it apparent, but we might change that in the future as well)
<img width="269" alt="Bildschirm­foto 2023-03-08 um 10 32 01" src="https://user-images.githubusercontent.com/94838646/223676021-356e7b69-0964-4507-bdc2-136e1180219f.png">

<img width="274" alt="Bildschirm­foto 2023-03-08 um 10 24 13" src="https://user-images.githubusercontent.com/94838646/223675675-71a62b2c-2ae0-49cd-bbfd-2967205b9a25.png">

Reply:
Ah, bummer. Good catch. Ye then lets keep it as is.